src/app/containers/SocialEmbed/index.jsx

```diff
@@ -49,6 +50,10 @@ const SocialEmbedContainer = ({ blocks }) => {
   if (!href) return null;
 
   const oEmbed = path(['embed', 'oembed'], model);
+  const maxHeight = path(
+    ['embed', 'fallback_image', 'fallback_image_height'],
```

Comment:
@simonsinclair yes that info is available in the `embed.oembed` block as seen [here](https://github.com/bbc/simorgh/blob/latest/data/ukrainian/cpsAssets/news-53561143.json#L4-L23) but it seems that's not always the case, [like here where the height is missing](https://github.com/bbc/simorgh/blob/latest/data/arabic/cpsAssets/23298105.json#L166-L181) (maybe it's just a mock data problem). The original issue stated to use the fallback values but if we're sure we'd always have height and width values in the `embed` object, I'd be happy to switch to that as well. @chris-hinds what do you think ?

Reply:
Thanks, Rowland.

- If a height value isn't provided by the Provider in `embed.oembed` how can we accurately rely upon any other value? In the example of the data with the missing `height` value - `width` _is_ provided by the provider, but it is different from the width in `fallback_image. fallback_image_width`. This is my worry with using values based on a screenshot/fallback image to restrict elements on the DOM.
- A compromise might be to restrict using the value IF there is one, otherwise do nothing OR combine with another fixed solution that is consistent.

cc @chris-hinds

Reply:
This is a good point both.

It looks like it would be better to use the values from the provider if possible.

Also will setting the min height help? as we could always have a default min height of say 300px to help reserve some space for when we don't have the height value